ui/components/FilterSummary/index.js

```diff
@@ -78,7 +78,7 @@ export function FilterSummary({ schema }) {
                 ) : null}
                 {filters.map((filter, i) => {
                   const connector = (
-                    (filtersSchema[key] && filtersSchema[key].type) ||
+                    (i > 0 && filtersSchema[key] && filtersSchema[key].type) ||
```

Comment:
I'm not sure what the filtersSchema is giving us in this scenario now. Is it silly to just put e.g.

```suggestion
                    (i > 0 && 'or' || 'and') 
```